internal/utils/ci_providers.go

```diff
@@ -277,18 +285,33 @@ func extractGithubActions() map[string]string {
 		branch = branchOrTag
 	}
 
-	url := fmt.Sprintf("https://github.com/%s/commit/%s/checks", os.Getenv("GITHUB_REPOSITORY"), os.Getenv("GITHUB_SHA"))
+	serverUrl := os.Getenv("GITHUB_SERVER_URL")
+	if serverUrl == "" {
+		serverUrl = "https://github.com"
+	}
+
+	rawRepository := fmt.Sprintf("%s/%s", serverUrl, os.Getenv("GITHUB_REPOSITORY"))
```

Comment:
this is not save if they provide `GITHUB_SERVER_URL` with a leading slash

Reply:
fixed in 334a193